### PR TITLE
Update cfpb-chart-builder dependency

### PR DESCRIPTION
**Pull request by contolini**
## Changes

- Bumped `cfpb-chart-builder` to v1.2.0.

## Checklist

* [x] PR has an informative and human-readable title
* [x] Changes are limited to a single goal (no scope creep)
* [x] Code can be automatically merged (no conflicts)
* [x] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [x] Passes all existing automated tests
* [x] New functions include new tests
* [x] New functions are documented (with a description, list of inputs, and expected output)
* [x] Placeholder code is flagged
* [x] Visually tested in supported browsers and devices
* [x] Project documentation has been updated
* [x] Reviewers requested with the [Assignee tool](https://help.github.com/articles/assigning-issues-and-pull-requests-to-other-github-users/) :arrow_right:

### CODE REVIEW

File: npm-shrinkwrap.json [hunk 1]
```diff
@@ -1833,6 +2720,706 @@
       "from": "fs.realpath@>=1.0.0 <2.0.0",
       "resolved": "https://registry.npmjs.org/fs.realpath/-/fs.realpath-1.0.0.tgz"
     },
+    "fsevents": {
```
Comment (willbarton) [inline npm-shrinkwrap.json]: 😡

Comment (willbarton) [inline npm-shrinkwrap.json]: We added `./frontend.sh shrinkwrap` to hopefully make this less likely.

Comment (contolini) [inline npm-shrinkwrap.json]: brb running `./frontend.sh shrinkwrap`

Comment (willbarton) [inline npm-shrinkwrap.json]: It is of critical importance that we develop an emoji to express our feelings about shrinkwrap.

Comment (willbarton) [inline npm-shrinkwrap.json]: 5z6on9j

**Pull request merged**